cli-plugins/manager/cobra.go

```diff
@@ -46,14 +46,8 @@ func AddPluginCommandStubs(dockerCli command.Cli, rootCmd *cobra.Command) (err e
 		}
 		for _, p := range plugins {
 			p := p
```

Comment:
Actually just now looking at the code changes; I don't think we need to change the code here. It may be simpler than that; probably only needs the go template to be updated (the one used to print the help output); see this line (and some lines below); https://github.com/docker/cli/blob/efb9206433fdb9a24e459d0f7b7cfc4e65b9b14b/cli/cobra.go#L473

Reply:
oh wow, i missed that. I'll change the code then